sdk/aqueduct/artifacts/numeric_artifact.py

```diff
@@ -110,8 +110,6 @@ def get(self, parameters: Optional[Dict[str, Any]] = None) -> Number:
             if parameters is None and self._get_content() is None:
                 self._set_content(content)
 
-            return content
```

Comment:
Do we only have to do this for numeric artifact class?

Reply:
yep